Tasks/Common/nuget-task-common/NuGetConfigHelper2.ts

```diff
@@ -161,16 +161,16 @@ export class NuGetConfigHelper2 {
                     if (configXml.packages) {
                         throw new Error(tl.loc(
                             "NGCommon_NuGetConfigIsPackagesConfig",
-                            this.nugetConfigPath,
+                            this.nugetConfigPath || this.tempNugetConfigPath,
```

Comment:
Right, the ordering is different though. In the messages I want to give priority to nugetConfigPath if existent whereas when reading it I want to give priority to the temp config.